variables.tf

```diff
@@ -676,42 +676,42 @@ variable "logging_use_private_endpoint" {
 variable "install_monitoring_agent" {
   type        = bool
   default     = false
-  description = "Set to true to enable installing the monitoring agent into your VSI at time of creation."
+  description = "Set to true to install the agent for monitoring, or for security and compliance of your VSI."
 }
 
-variable "monitoring_access_key" {
+variable "access_key" {
```

Comment:
I would change this back what it was so its clear the access key is for monitoring